huaweicloud/provider.go

```diff
@@ -1127,6 +1127,7 @@ func Provider() *schema.Provider {
 			"huaweicloud_dataarts_architecture_data_standard":          dataarts.ResourceDataStandard(),
 			"huaweicloud_dataarts_architecture_data_standard_template": dataarts.ResourceDataStandardTemplate(),
 			"huaweicloud_dataarts_architecture_reviewer":               dataarts.ResourceDataArtsArchitectureReviewer(),
+			"huaweicloud_dataarts_architecture_code_table_values":      dataarts.ResourceArchitectureCodeTableValues(),
```

Comment:
suggest to move this line after `huaweicloud_dataarts_architecture_code_table` resource

Reply:
fiexed